valhalla/jawn/src/lib/stores/OnlineEvalStore.ts

```diff
@@ -0,0 +1,88 @@
+import { RequestResponseRMT, clickhouseDb } from "../db/ClickhouseWrapper";
+import { dbExecute } from "../shared/db/dbExecute";
+import {
+  err,
+  resultMap,
+  ok,
+  Result,
+  PromiseGenericResult,
+} from "../shared/result";
+import { BaseStore } from "./baseStore";
+
+export type OnlineEvaluatorByOrgId = {
+  id: string;
+  evaluator_id: string;
+  evaluator_scoring_type: string;
+  evaluator_name: string;
+  evaluator_llm_template: string;
+  config: any;
+};
+
+export type OnlineEvaluatorByEvaluatorId = {
+  id: string;
+  config: any;
+};
+
+export class OnlineEvalStore extends BaseStore {
```

Comment:
It'd be nice, if we can clean this up and use abstractions instead of Copy Pasta

Reply:
not necessary if there is too much small changes and vars to pull out